cosmos/settings.py

```diff
@@ -66,3 +66,5 @@ def convert_to_boolean(value: str | None) -> bool:
 enable_telemetry = conf.getboolean("cosmos", "enable_telemetry", fallback=True)
 do_not_track = convert_to_boolean(os.getenv("DO_NOT_TRACK"))
 no_analytics = convert_to_boolean(os.getenv("SCARF_NO_ANALYTICS"))
+
+_AIRFLOW3_VERSION = Version("3.0.0a1")
```

Comment:
This is a constant, not a user-configurable value, so it shouldn't be in `settings.py`. Should we move it to `constants.py`?

Also, why `a1`? Probably `Version("3")` is more accurate

Reply:
Yes, constants.py would be the right place for this. Thanks!

> Also, why a1? Probably Version("3") is more accurate

Since we have just the alpha/beta version available at the moment, if we will keep `Version("3")` it would cause a problem in dev env because `3.0.0b2 < 3`

```
>>> v1 = Version("3.0.0b2")
>>> v2 = Version("3")
>>> v1 < v2
True
```